src/main/java/edu/oregonstate/cs361/battleship/Main.java

```diff
@@ -592,22 +592,22 @@ else if( posHelper(model.getDinghy(), mycoord )){
     private static Coord AIHardFire(BattleshipModel model) {
         Random rand = new Random(System.currentTimeMillis());
         Coord mycoord;
-        //If the previious shot was not a hit
+        //If the previious shot was not a hit, shoot randomly
         do {
             mycoord = new Coord(rand.nextInt(boardHeight+1), rand.nextInt(boardWidth+1));
 
         } while (!checkValidShot(model, mycoord));//while the shot has already been done
 
         //If it was a hit
-
+        //Shot the areas that are around the preious shot because that is likely to be another hit

```

Comment:
One thing that we have to make sure to figure out is whether the AI will be able to figure out which direction the ship is going in and keep shooting in that same line, or if it just randomly fires around its last hit. If we want it to just fire around the previous hit, should it just fire once nearby, then give up on that area? Or should it fire in all 4 directions around that area until it gets a hit. 
I think the best way to do this would be to just have it fire randomly around its last hit until it has exhausted all 4 directions or it hits something, then move onto the same thing for that new spot. I don't like the idea of the AI figuring out the orientation of the ship then firing along that line because that might make the AI too hard.